test/mocks/permissions.js

```diff
@@ -0,0 +1,369 @@
+import deepFreeze from 'deep-freeze-strict';
+import { CaveatTypes } from '../../shared/constants/permissions';
+
+/**
+ * This file contains mocks for the PermissionLogController tests.
+ */
+
+export const noop = () => undefined;
+
+const keyringAccounts = deepFreeze([
+  '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+  '0xc42edfcc21ed14dda456aa0756c153f7985d8813',
+  '0x7ae1cdd37bcbdb0e1f491974da8022bfdbf9c2bf',
+  '0xcc74c7a59194e5d9268476955650d1e285be703c',
+]);
+
+const SUBJECTS = {
+  a: { origin: 'https://foo.xyz', host: 'foo.xyz' },
+  b: { origin: 'https://bar.abc', host: 'bar.abc' },
+  c: { origin: 'https://baz.def', host: 'baz.def' },
```

Comment:
We don't use the `host` anywhere
```suggestion
  a: { origin: 'https://foo.xyz' },
  b: { origin: 'https://bar.abc' },
  c: { origin: 'https://baz.def' },
```